godtools/App/Features/Menu/Domain/UseCases/GetUserActivityStatsUseCase/GetUserActivityStatsUseCase.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GetUserActivityStatsUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 3/20/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import GodToolsToolParser
+import SwiftUI
+
+class GetUserActivityStatsUseCase {
+    
+    private let localizationServices: LocalizationServices
+    
+    init(localizationServices: LocalizationServices) {
+        
+        self.localizationServices = localizationServices
+    }
+    
+    func getUserActivityStats(from userActivity: UserActivity) -> [UserActivityStatDomainModel] {
+        
+        let toolOpensStat = UserActivityStatDomainModel(
+            iconImageName: "tool-opens",
```

Comment:
Hey @rachaelblue for the image names can you place these in ```ImageCatalog.swift``` and pull from there.